autospec/pkg_integrity.py

```diff
@@ -498,8 +499,11 @@ def verify(self):
         package_info = info.get('info', None)
         if package_info is not None:
             EMAIL = package_info.get('author_email', '')
-        self.md5_digest = release_info.get('md5_digest', '')
-        return self.verify_md5()
+        sha256 = ''
```

Comment:
The verify function docs still say verify with MD5.

Reply:
thanks, I will update the docstring